### PR TITLE
[FIX] stock_landed_costs: landed Cost to be hidden on other product types

### PR DESCRIPTION
**Pull request by suth-odoo**
Steps to Reproduce :
- install sales, stock modules
- go to products and create new
- select product type as Event Ticket,Event Booth or Gift Card
- Landed cost boolean is suppose to be visible for a service type product under purchase.

Issue :
- Landed cost boolean is suppose to be visible for a service type product under purchase, But currently it is visible on other type of products specific to other models like event tickets, event booths etc.

Cause:
- In product Module there are two Selection fields like detailed_type and type. The detailed_type Selection field contain service, consumable, event booths, event tickets, storable product  And the type selection field contains service, consumable, storable product
- In xml views for that landed cost boolean the invisible  attribute contains the condition like  type != service, because of this the landed cost boolean is visible for the product type of event tickets, event booths and gift Cards.

Solution:
- if we use detailed_type instead of type selection field, then the landed cost boolean field  will be invisible on other product types.

 task- 3725202

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by pkyriakou — COMMENTED** [review]
Thank you @suth-odoo :100: I don't have anything to add, LGTM :green_circle: :top:

**Review by auon-odoo — APPROVED** [review]

**Review by xavierbol — COMMENTED** [review]
Hello @suth-odoo 

Thanks for your work! 

However the commit title does not fit our guidelines, the module in which you did the fix is `stock_landed_costs` and not `sales`.

Also, in the steps to reproduce, you say install `sales` but only with that module, the `stock_landed_costs` will not be installed as well. We should **also** install stock module and so with `sale` and `stock` modules, `stock_landed_costs` will normally automatically installed.

Can you update that and rebase your branch at the same time? 🙂 

Thanks in advance!

**Comment by suth-odoo** [comment]
Hello @xavierbol 
Thanks for your review... I updated the commit ..check once if free

**Review by xavierbol — APPROVED** [review]

**Comment by xavierbol** [comment]
Hello @odoo/rd-logistics 

Can we have your opinion about that fix before merging it? 

Thanks in advance! 🙂

**Comment by xavierbol** [comment]
robodoo r+

**Pull request closed without merging**
